helmcharts/services/alloy/CHANGELOG.md

```
@@ -0,0 +1,369 @@
+# Changelog
+
+> _Contributors should read our [contributors guide][] for instructions on how
```

markdownlint-cli2 (0.20.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "contributors guide"

(MD052, reference-links-images)